Header header
string goal_id
float64 bid
